src/scenes/AccessionsRouter/Accession2CreateView.tsx

```diff
@@ -227,6 +229,7 @@ export default function CreateAccession(): JSX.Element {
                 record={record}
                 setRecord={setRecord}
                 availableProjects={availableProjects}
+                allowUnselect
```

Comment:
Do we need a similar option for accession edit?

Reply:
I'm not really sure, it seems like it should be though. There isn't any mention of that in the designs, and the PRD technically only [mentions](https://terraformation.atlassian.net/wiki/spaces/PM/pages/811303013/PRD+Projects#As-a-manager-or-an-admin-of-an-organization-I-want-to-add%2Fchange-a-project-of-an-existing-accession%2Fseedling-batch%2Fplanting-site) "add or change a project".

Reply:
Got it, maybe a question for @chudo76

Reply:
Doing this in a follow up PR now